doc/classes/EditorTranslationParserPlugin.xml

```diff
@@ -106,14 +106,50 @@
 		<method name="_get_recognized_extensions" qualifiers="virtual const">
 			<return type="PackedStringArray" />
 			<description>
-				Gets the list of file extensions to associate with this parser, e.g. [code]["csv"][/code].
+				Override this method to provide the list of file extensions to associate with this parser, e.g. [code]["csv"][/code]. See also [method get_recognized_extensions].
 			</description>
 		</method>
 		<method name="_parse_file" qualifiers="virtual">
 			<return type="PackedStringArray[]" />
 			<param index="0" name="path" type="String" />
 			<description>
-				Override this method to define a custom parsing logic to extract the translatable strings.
+				Override this method to define a custom parsing logic to extract the translatable strings. See also [method parse_file].
+			</description>
+		</method>
+		<method name="get_all_recognized_extensions" qualifiers="static">
+			<return type="PackedStringArray" />
+			<description>
+				Returns a list of all file extensions supported by any translation parser, both standard and custom.
+			</description>
+		</method>
+		<method name="get_builtin_strings" qualifiers="static">
+			<return type="PackedStringArray[]" />
+			<description>
+				Returns a list of strings from built-in components such as certain [Control] nodes in the format [code][msgid, msgctxt, msgid_plural][/code].
+			</description>
+		</method>
+		<method name="get_parser" qualifiers="static">
+			<return type="EditorTranslationParserPlugin" />
+			<param index="0" name="extension" type="String" />
+			<description>
+				Returns a [EditorTranslationParserPlugin] associated with the file extension [param extension]. Useful if you want to intercept a standard parser in your editor tool.
+				[codeblock]
+				var parser = EditorTranslationParserPlugin.get_parser("gd")
+				print(parser.parse_file("res://your_script.gd"))
+				[/codeblock]
+			</description>
+		</method>
+		<method name="get_recognized_extensions" qualifiers="const">
+			<return type="PackedStringArray" />
+			<description>
+				Returns a list of file extensions associated with this parser, e.g. [code]["csv"][/code]. See also [method _get_recognized_extensions].
+			</description>
+		</method>
+		<method name="parse_file">
+			<return type="PackedStringArray[]" />
+			<param index="0" name="path" type="String" />
+			<description>
+				Returns a list of translatable strings extracted from the file [param path]. See also [method _parse_file].
```

Comment:
Neither this method's description nor the virtual method describe the format of the return value (I assume it is the same as the builtin string list?). It'd be nice to have it documented.

Reply:
This is documented in the class description.